src/main/java/reiff/projectile/Projectile.java

```diff
@@ -0,0 +1,37 @@
+package reiff.projectile;//TIP To <b>Run</b> code, press <shortcut actionId="Run"/> or
+
+import static java.lang.Math.cos;
+import static java.lang.Math.sin;
+
+// click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
+public class Projectile {
+
+    private double angle;
+    private double radians;
+    private double velocity;
+    private double seconds;
+    private double apexTime;
+
+    public Projectile(double angle, double velocity, double seconds) {
```

Comment:
Seconds shouldn't be part of the constructor. Make a separate method to set the seconds